lib/avalon/ffprobe.rb

```diff
@@ -23,7 +23,9 @@ def json_output
       return @json_output unless @json_output.nil?
       return @json_output = {} unless valid_content_type?(@media_file)
       ffprobe = Settings&.ffprobe&.path || 'ffprobe'
-      raw_output = `#{ffprobe} -i "#{@media_file.location}" -v quiet -show_format -show_streams -of json`
+      # Include authorization headers for cases like Google Drive
+      header = "-headers 'Authorization: #{@media_file.auth_header.fetch('Authorization')}'" if @media_file.auth_header.present?
+      raw_output = `#{ffprobe} #{header} -i "#{@media_file.location}" -v quiet -show_format -show_streams -of json`
```

Comment:
This is downloading the file for processing, but we have already downloaded the file as part of the [`valid_content_type?` check](https://github.com/avalonmediasystem/avalon/blob/ecb7995e42339a9e35cf5a8c9d78336c03814c48/lib/avalon/ffprobe.rb#L79). 

It would probably be best as a separate ticket, but should we rework this so that when we are trying to access an internet hosted file we download to a tempfile and use the tempfile for all the processing instead of downloading it multiple times?